doc/tutorials/introduction/windows_install/windows_install.markdown

```diff
@@ -24,6 +24,84 @@ Installation by Using the Pre-built Libraries {#tutorial_windows_install_prebuil
 
 -#  To finalize the installation go to the @ref tutorial_windows_install_path section.
 
+Installation by Using git-bash (version>=.14.1) and cmake (version >=3.9.1)
+===============================================================
+
+-#  You must download [cmake (version >=3.9.1)](https://cmake.org) and install it. You must add cmake to PATH variable during installation
+
+-#  You must install [git-bash (version>=.14.1)](https://git-for-windows.github.io/). Don't add git to PATH variable during installation
+
+-#  Run git-bash. You observe a command line window.
+Suppose you want to build opencv and opencv_contrib in c:/lib
+
+-#  In git command line enter following command (if folder does not exist) :
+@code{.bash}
+mkdir /c/lib
+cd /c/lib
+@endcode
+
+-#  save this script with name installOCV.sh in c:/lib
+
+        @code{.bash}
+myRepo=$(pwd)
+CMAKE_CONFIG_GENERATOR="Visual Studio 14 2015 Win64"
+if [  ! -d "$myRepo/opencv"  ]; then
+  echo "clonning opencv"
+  git clone https://github.com/opencv/opencv.git
+  mkdir Build
+  mkdir Build/opencv
+  mkdir Install
+  mkdir Install/opencv
+else
+  cd opencv
+  git pull
+  cd ..
+fi
+if [  ! -d "$myRepo/opencv_contrib"  ]; then
+  echo "clonning opencv_contrib"
+  git clone https://github.com/opencv/opencv_contrib.git
+  mkdir Build
+  mkdir Build/opencv_contrib
+else
+  cd opencv_contrib
+  git pull
+  cd ..
+fi
+RepoSource=opencv
+cd Build/$RepoSource
+optCMAKE='-DBUILD_PERF_TESTS:BOOL=OFF -DBUILD_TESTS:BOOL=OFF -DBUILD_DOCS:BOOL=OFF  -DWITH_CUDA:BOOL=OFF -DBUILD_EXAMPLES:BOOL=OFF'
+cmake -G"$CMAKE_CONFIG_GENERATOR" $optCMAKE -DOPENCV_EXTRA_MODULES_PATH="$myRepo"/opencv_contrib/modules -DCMAKE_INSTALL_PREFIX="$myRepo"/install/"$RepoSource" "$myRepo/$RepoSource"
+cd ..
+echo "************************* $Source_DIR -->debug"
+cmake --build $RepoSource  --config debug
+echo "************************* $Source_DIR -->release"
+cmake --build $RepoSource  --config release
+cmake --build $RepoSource  --target install --config release
+cmake --build $RepoSource  --target install --config debug
```

Comment:
> --target install

Consider to use `-DINSTALL_CREATE_DISTRIB=ON` CMake option to prevent touching of global files (like Python module from Python subfolder)

Reply:
OK